src/sisyphus/cloud.clj

```diff
@@ -269,3 +277,52 @@
             relative-path (.substring remote-key prefix-length)
             local-path (join-path [path relative-path])]
         (download-blob! blob local-path)))))
+
+(defn create-compute-service
+  "Create an instance of com.google.api.services.compute.Compute to make requests with."
+  []
+  (let [transport (GoogleNetHttpTransport/newTrustedTransport)
+        factory (JacksonFactory/getDefaultInstance)
+        auth (into-array ["https://www.googleapis.com/auth/cloud-platform"])
+        credential (GoogleCredential/getApplicationDefault)
+        credential (if (.createScopedRequired credential)
+                     (.createScoped credential (Arrays/asList auth))
+                     credential)
+        builder (Compute$Builder. transport factory credential)]
+    (.setApplicationName builder "Gaia/0.0.1")
+    (.build builder)))
```

Comment:
Is this a case where it's good practice to declare the result type?

Reply:
Yeah, makes sense. I'll add it in.